app/views/confirm.scala.html

```diff
@@ -24,12 +24,13 @@
 @import org.joda.time.DateTime
 @import _root_.template.FormatRefNum
 @import models.journeys.Journey
-
+@import config.ForConfig
 
 @(theForm: Form[_], referenceNumber: String, email: Option[String], summary: Summary)(implicit requestHeader: RequestHeader,  messages: Messages)
 
 @headExtra = {
   @views.html.includes.stylesheets()
+  @views.html.includes.analytics(ForConfig.config.getString("analytics.ga.GTM").getOrElse(""), summary)
```

Comment:
can we replace `ForConfig.config.getString("analytics.ga.GTM").getOrElse("")` with `ForConfig.gtm` and move original code to `ForConfig`

Reply:
Code change has been made and committed.